### PR TITLE
Install DeploySentinel Playwright reporter

### PR DESCRIPTION
**Pull request by wrn14897**
## What does this PR do?

DeploySentinel is a dev-friendly reporting tool built for Playwright

#### How it works

* Save Time & Sanity Fixing Tests in CI

image

image

* Get a birds-eye view on errors, flakes, slow tests, and common error messages all within a few clicks

image

[More details](https://www.deploysentinel.com/playwright)

**Environment**: CI

## Type of change

- [ ] Chore (refactoring code, technical debt, workflow improvements)

## How should this be tested?

- [ ] Test A
- [ ] Test B

### CODE REVIEW

File: .github/workflows/e2e-app-store.yml [hunk 1]
```diff
@@ -93,6 +93,8 @@ jobs:
         run: yarn playwright install --with-deps
       - name: Run Tests
         run: yarn test-e2e:app-store
+        env:
+          DEPLOYSENTINEL_API_KEY: ${{ secrets.DEPLOYSENTINEL_API_KEY }}
```
Comment (wrn14897) [inline .github/workflows/e2e-app-store.yml:97]: The api key can be found [here](https://www.deploysentinel.com/team) after signup

**Comment by wrn14897** [comment]
It looks like 'PR Update' workflows are not happy with yarn.lock changes. I checked out  yarn.lock from the main and reinstalled the dep and still couldn't get it working (tried installing both at root and `/apps/web` ). Would be nice to get some helps from the core team :pray:  @zomars

**Comment by zomars** [comment]
Hey @wrn14897 Can you enable "allow edits by maintainers" so I can update this PR before merging?

**Comment by wrn14897** [comment]
> Hey @wrn14897 Can you enable "allow edits by maintainers" so I can update this PR before merging?

Hey zomars, thanks for checking it out. I dont see the "allow edits by maintainers" checkbox on my end so I assume you should be able to push changes. Lmk if there is anything else I can do

**Comment by zomars** [comment]
> > Hey @wrn14897 Can you enable "allow edits by maintainers" so I can update this PR before merging?
> 
> Hey zomars, thanks for checking it out. I dont see the "allow edits by maintainers" checkbox on my end so I assume you should be able to push changes. Lmk if there is anything else I can do

I'm getting an unauthorized error when trying to push 🤔

**Comment by wrn14897** [comment]
> > > Hey @wrn14897 Can you enable "allow edits by maintainers" so I can update this PR before merging?
> > 
> > 
> > Hey zomars, thanks for checking it out. I dont see the "allow edits by maintainers" checkbox on my end so I assume you should be able to push changes. Lmk if there is anything else I can do
> 
> I'm getting an unauthorized error when trying to push thinking

Sorry for the inconvenience. It looks like gh doesnt support that option for org repo. I duplicated pr with personal account. This PR should work https://github.com/calcom/cal.com/pull/7405

**Comment by zomars** [comment]
Closing in favor of https://github.com/calcom/cal.com/pull/7405

**Pull request closed without merging**